src/index.tsx

```diff
@@ -1191,6 +1191,47 @@ function getSelectedBackgroundColor() {
   return backgroundColors.length === 1 ? backgroundColors[0] : null;
 }
 
+function addTextElement(element: ExcalidrawElement) {
+  if (isTextElement(element)) {
+    resetCursor();
+    const text = prompt("What text do you want?");
+    if (text === null) {
+      return;
```

Comment:
If you put it in a function, then it will change the semantics, it won't return from the outer one anymore :(